packages/abap-deploy-config-inquirer/src/prompts/questions/config/package.ts

```diff
@@ -87,9 +91,15 @@ export function getPackagePrompts(
                 breadcrumb: true
             },
             default: (previousAnswers: AbapDeployConfigAnswersInternal): string =>
-                defaultPackage(previousAnswers.packageManual || options.packageManual?.default),
+                defaultPackage(previousAnswers.packageManual || options.packageManual?.default, options?.packageManual),
```

Comment:
I saw the [SonarQube code smell](https://sonarcloud.io/project/issues?issueStatuses=ACCEPTED&pullRequest=2603&id=SAP_open-ux-tools&open=AZWGgx2SSqUDach4NQf4) for this line and read the comment from @cianmSAP:
"Empty strings may be valid". 
Don't fully get this, if we use `||` an empty string would be `false` and `options.packageManual?.default` is used.
If a string value is fine, I would expect to use `??` and the code smell would be valid and have to  be fixed. Otherwise, if empty string is a valid value, I'd propose to do:
```suggestion
                defaultPackage(typeof previousAnswers.packageManual === 'string' ? previousAnswers.packageManual : options.packageManual?.default, options?.packageManual),
```
to make it clear.

Reply:
Sorry, my comment was innacurate. 
The issue with `??` specifically with yeoman is that if a question is not answered it still resolves the answer to be an empty string, rather than undefined. So we have encountered issues where the empty string is either used for defaults, or answers, when it should not.